scripts/components/statssynchronizercomponent/StatsSynchronizerComponent.gd

```diff
@@ -291,12 +312,15 @@ func add_experience(amount: int):
 
 
 func apply_boost(boost: Boost):
-	hp_max += boost.hp_max
-	hp += boost.hp
-	attack_power_min += boost.attack_power_min
-	attack_power_max += boost.attack_power_max
+	for statName in boost.statBoostDict:
+		assert(get(statName) != null, "This property does not exist as a stat.")
```

Comment:
I don't think this check should be in an assert but in an if statement with error handling to have it also present on released builds.